fossunited/patches/v1_0/duplicate_event_name_to_reference_field.py

```diff
@@ -0,0 +1,39 @@
+import frappe
+
+
+def execute():
+    """
+    In this patch, we will be going through every existing Email Groups, Email Templates,
+    and Newsletters & we will be copying the value of the `event` field to the
+    `reference_document` field of the same document.
+    """
+    email_templates = frappe.get_all(
+        "Email Template",
+        filters={"event": ("!=", "")},
+        fields=["name", "event"],
+    )
+    for email_template in email_templates:
+        frappe.db.set_value(
+            "Email Template",
+            email_template.name,
+            "reference_document",
+            email_template.event,
+        )
```

Comment:
shouldn't we also be setting `document_Type` for all of them or is it not necessary because of the default that is listed in `fossunited.setup`?

Reply:
Default is set to be `FOSS Chapter Event`, and we don't have any hackathon related groups rn so it will be fine. It will auto set it to be FOSS Chapter Event